.github/workflows/make_release.yaml

```diff
@@ -19,7 +19,9 @@ jobs:
     - name: Bump version
       run: |
         i=${{github.event.inputs.versionNumber}}
-        [[ "$i" == '0' ]] || [[ "$i" == '1' ]] || [[ "$i" == '2' ]] || (echo "index must be 0, 1, or 2"; exit 1)
+        if [[ "$i" =~ [^0-9] ]] || (( i < 0 || i > 2 )); then
```

Comment:
Actually, why don't we simply check for the three explicit single digit values we allow?
```bash
if ! [[ "$i" =~ ^[012]$ ]]; then 
    echo "index must be 0, 1, or 2"; exit 1;
fi
```

Reply:
… not that it matters in any practical way, shape or form … :sweat_smile:

Reply:
Changed